doc/remotes.md

```diff
@@ -0,0 +1,52 @@
+# Remotes
+
+## Introduction
+
+Remotes (machine with LXD installed) is concept used by LXD through which users doesn't have to always provide hostname or IP and then validating certificate information.
+LXD introduce three types of remotes:
+
+- Built-in
+- System
+- User
+
+### Built-in
+
+Built-in remotes are:
+- local (default)
+- images
+- ubuntu
+- ubuntu-daily
+
+Local, ubuntu, ubuntu-daily are static and they can't be modified by user.
+
+### System
+
+System remotes can be managed only manually. Remote configuration path is: `/etc/lxd/config/config.yml`.
+System remotes certifications should be placed in `/etc/lxd/config/servercerts/` and named after remote name e.g. `foo.crt`
```

Comment:
Should be:
"System remote certificates should be placed in `/etc/lxd/servercerts/` and be named after their respective remote. e.g. `foo.crt`"